composer.json

```diff
@@ -20,27 +20,27 @@
     "require": {
         "php": "^5.5.9 || ~7.0",
         "twig/extensions": "^1.0",
-        "symfony/symfony": "^3.3",
+        "symfony/symfony": "^2.8.7 || ^3.0",
         "symfony/monolog-bundle": "^2.8.7 || ^3.0",
-        "sensio/distribution-bundle": "^5.0",
+        "sensio/distribution-bundle": "^5.0.6",
         "sensio/framework-extra-bundle": "^3.0.2",
-        "incenteev/composer-parameter-handler": "^2.0",
-        "sulu/sulu": "~1.6.0",
+        "incenteev/composer-parameter-handler": "~2.1",
+        "sulu/sulu": "dev-master as 1.6.16",
+        "sulu/document-manager": "dev-master as 0.10.1",
         "dantleech/phpcr-migrations-bundle": "~1.0",
         "zendframework/zend-stdlib": "^2.3",
         "zendframework/zendsearch": "@dev",
         "massive/build-bundle": "0.3.*",
         "jackalope/jackalope-doctrine-dbal": "^1.2.5",
-        "jackalope/jackalope-jackrabbit": "^1.2.0",
+        "jackalope/jackalope-jackrabbit": "^1.2",
         "doctrine/doctrine-fixtures-bundle": "~2.3",
         "doctrine/doctrine-bundle": "^1.0",
-        "doctrine/doctrine-cache-bundle": "^1.0",
-        "oro/doctrine-extensions": "^1.0"
```

Comment:
@sulu/core-team need this be required by sulu-minimal?

Reply:
i would say no ...